package.json

```diff
@@ -42,6 +42,7 @@
     "@react-navigation/stack": "7.1.0",
     "@rneui/base": "4.0.0-rc.8",
     "@sentry/react-native": "6.4.0",
+    "@shopify/flash-list": "1.7.3",
```

Comment:
Thank you for adding FlashList.  I wanted to use it in other places to see performance gain.  I see a lot of VirtualizedList error every once in a while when scrolling through pages and pages of posts, and always thought this could be improved with FlashList.

Reply:
Tried.. is a huge pain because the culprit there is the post component being super heavy, but please please give it a try my friend.. it opens up a lot of possibilities